README.md

```diff
@@ -148,7 +148,11 @@ rustup target add riscv32i-unknown-none-elf
 
 ### Rust with Podman or Docker
 
-Alternatively you might build the project in the container where image already contains pre-installed Rust and ESP-IDF.
+Alternatively you might build the project in the container where the image already contains pre-installed Rust and ESP-IDF.
+
+There are two different images published to Dockerhub: 
+- ![idf-rust](https://hub.docker.com/r/espressif/idf-rust) - contains only the toolchain.
+- ![idf-rust-examples](https://hub.docker.com/r/espressif/idf-rust-examples) - includes example code from https://github.com/espressif/rust-esp32-example and is about 1GB larger in size. 
```

Comment:
Links are not working properly due to the `!` at the start, they are being treated as images.

Reply:
Oops, dumb mistake 😬